spyder/widgets/mixins.py

```diff
@@ -22,9 +22,11 @@
 from qtpy.QtCore import QPoint, Qt
 from qtpy.QtGui import QCursor, QTextCursor, QTextDocument
 from qtpy.QtWidgets import QApplication, QToolTip
-from qtpy import PYQT5
+from qtpy import PYQT_VERSION
 
-if PYQT5:
+PYQT_VERSION = float(PYQT_VERSION)
+

```

Comment:
hmm no, It's given an error due versions like 4.11.4

I'll implement:
`MINOR_VERSION = int(PYQT_VERSION.split('.')[1])`